### DISCUSSION TITLE
请问在官方文档的首页顶部的旋转logo和渐变文字是如何做到的?

### DISCUSSION THREAD

**Discussion by surkaa** [vuepress-theme-hope/vuepress-theme-hope#3940] [Q&A] [answered]
[官网](https://theme-hope.vuejs.press/zh/)
![image](https://github.com/vuepress-theme-hope/vuepress-theme-hope/assets/98200894/da0b6926-808a-4a44-907f-cffe9d9d7e68)

**Comment by Mister-Hope** [comment, marked as answer]
这是个开源项目，意味着你可以查阅任何源码，而且文档已经足够啰嗦的教你如何自定义了。
